lib/utils.js

```diff
@@ -336,7 +336,7 @@ function sanitizeQuery(query, options) {
   const prohibitedKeys = options.prohibitedKeys;
   const offendingKeys = [];
   const normalizeUndefinedInQuery = options.normalizeUndefinedInQuery;
-  const maxDepth = options.maxDepth || 12;
+  const maxDepth = options.maxDepth || Number.MAX_SAFE_INTEGER;
```

Comment:
Is it a good idea to allow arbitrary deep queries? Or is this reverting back to the old behavior we used to have in juggler before recent security-related fixes?

Reply:
This is a default value for the utility method itself. Juggler already sets `32` as the default value.

Reply:
So can we use `32` as the default value in this place too? I'd rather avoid all possibilities of implicitly infinite depth.